### DISCUSSION TITLE
Different between ghc & docker.pkg.github.com

### DISCUSSION THREAD

**Discussion by mittalyashu** [community/community#26890] [Actions]
<p>I tried to read the docs, it’s still bit confusing.</p>

**Comment by clarkbw** [comment, marked as answer]
<p>The announcement post does a good job explaining <a href="https://github.blog/2020-09-01-introducing-github-container-registry/">https://github.blog/2020-09-01-introducing-github-container-registry/</a></p>
<p>GHCR is the new docker.pkg.github</p>

**Comment by mittalyashu** [comment]
<p>When I try to publish a package from repository it still shows older version.</p>
<p>Whereas organisation packages tab shows newer version.</p>
<p>This difference, is it intentional?</p>

**Comment by clarkbw** [comment]
<p>Fix for that is coming <a href="https://github.community/t/github-container-registry-package-not-showing-on-repo/132611/2" class="inline-onebox">GitHub container registry - Package not showing on repo</a></p>
